command/src/main/java/com/jagrosh/jdautilities/command/MessageContextMenu.java

```diff
@@ -58,29 +58,30 @@ public final void run(MessageContextMenuEvent event)
         if(event.isFromGuild())
         {
             //user perms
-            for(Permission p: userPermissions)
-            {
-                // Member will never be null because this is only ran in a server (text channel)
-                if(event.getMember() == null)
-                    continue;
-
-                if(p.isChannel())
+            if (alwaysRespectUserPermissions)
```

Comment:
Given the size of the if-section here, this should probably be wrapped in {} for readability.